lineage/tHMM_utils.py

```diff
@@ -159,22 +159,17 @@ def getAIC(tHMMobj, LL):
     numStates = tHMMobj.numStates
     AIC_value_holder = []
     AIC_degrees_of_freedom_holder = []
+    number_of_parameters = 0
     for num in range(tHMMobj.numLineages):
-        number_of_parameters = 0
-        if tHMMobj.keepBern:
-            number_of_parameters += 1
-        if tHMMobj.FOM == 'G':
-            number_of_parameters += 2
-        elif tHMMobj.FOM == 'E':
-            number_of_parameters += 1
-
-        AIC_degrees_of_freedom = numStates**2 + numStates * number_of_parameters - 1
-        AIC_degrees_of_freedom_holder.append(AIC_degrees_of_freedom)
-        AIC_value = -2 * LL[num] + 2 * AIC_degrees_of_freedom
-        AIC_value_holder.append(AIC_value)
+        number_of_parameters += 1
+
+    AIC_degrees_of_freedom = numStates**2 + numStates * number_of_parameters - 1
```

Comment:
Still doesn't make sense, sorry. Why is the number of parameters dependent on the number of lineages?

Reply:
There's a T matrix which is numStates squared.
There's pi which is just numStates.
Then for each state, we want to find a Bernoulli theta and exponential lambda (2), so numStates *2.

`AIC_degrees_of_freedom = numStates**2 + numStates * number_of_parameters` is fine, but `number_of_parameters` should be 2 **always**.

Reply:
Actually, just add likelihood to the graph (another line plotting the likelihood values) unless it proves to be too difficult with the current way of how you store the data. 

More importantly the number of parameters thing has to be fixed; it should be 2, unless I'm misunderstanding something.

Reply:
> Still doesn't make sense, sorry. Why is the number of parameters dependent on the number of lineages?

i am not sure. originally, i just set the number of parameters equal to `numStates**2` but someone (i think yourself) added the extra `numStates * number_of_parameters` where `number_of_parameters` is equal to `numlineages` ....should we delete the latter i have never seen it in a literature but it is spefici to our application...

Reply:
> There's a T matrix which is numStates squared.
> There's pi which is just numStates.
> Then for each state, we want to find a Bernoulli theta and exponential lambda (2), so numStates *2.
> 
> `AIC_degrees_of_freedom = numStates**2 + numStates * number_of_parameters` is fine, but `number_of_parameters` should be 2 **always**.

I agree lets just do AIC_degrees_of_freedom = numStates**2 + numStates * (E.shape[1]) - 1` although E is a list so i guess `len(E[0,:])` which is the number of variables  in the thmm